java/google/registry/rde/imports/XjcToEppResourceConverter.java

```diff
@@ -14,49 +14,23 @@
 
 package google.registry.rde.imports;
 
-import com.google.common.base.Joiner;
+import com.google.common.base.Charsets;
 import google.registry.model.EppResource;
+import google.registry.xjc.XjcXmlTransformer;
+import google.registry.xml.XmlException;
 import java.io.ByteArrayOutputStream;
-import java.util.Arrays;
-import javax.xml.bind.JAXBContext;
-import javax.xml.bind.JAXBElement;
-import javax.xml.bind.JAXBException;
-import javax.xml.bind.Marshaller;
 
 /**
  * Base class for Jaxb object to {@link EppResource} converters
  */
 public abstract class XjcToEppResourceConverter {
 
-  /** List of packages to initialize JAXBContext. **/
-  private static final String JAXB_CONTEXT_PACKAGES = Joiner.on(":")
-      .join(Arrays.asList(
-          "google.registry.xjc.contact",
-          "google.registry.xjc.domain",
-          "google.registry.xjc.host",
-          "google.registry.xjc.mark",
-          "google.registry.xjc.rde",
-          "google.registry.xjc.rdecontact",
-          "google.registry.xjc.rdedomain",
-          "google.registry.xjc.rdeeppparams",
-          "google.registry.xjc.rdeheader",
-          "google.registry.xjc.rdeidn",
-          "google.registry.xjc.rdenndn",
-          "google.registry.xjc.rderegistrar",
-          "google.registry.xjc.smd"));
-
-  /** Creates a {@link Marshaller} for serializing Jaxb objects */
-  private static Marshaller createMarshaller() throws JAXBException {
-    return JAXBContext.newInstance(JAXB_CONTEXT_PACKAGES).createMarshaller();
-  }
-
-  protected static byte[] getObjectXml(JAXBElement<?> jaxbElement) {
+  protected static byte[] getObjectXml(Object jaxbElement) {
     try {
-      Marshaller marshaller = createMarshaller();
       ByteArrayOutputStream bout = new ByteArrayOutputStream();
-      marshaller.marshal(jaxbElement, bout);
+      XjcXmlTransformer.marshalLenient(jaxbElement, bout, Charsets.UTF_8);
```

Comment:
This version of Charsets is deprecated. I've switched it, and others like it, to use StandardCharsets.